src/main/java/com/softserve/academy/spaced/repetition/repository/CategoryRepository.java

```diff
@@ -19,4 +18,6 @@
 
     @Query(value = "SELECT c.name, c.description, c.image FROM Category c WHERE c.id = :categoryId")
     List<Category> hasAccessToCategory(@Param("categoryId") Long categoryId);
+
+    List<Category>  findByNameIgnoreCaseContainingOrDescriptionIgnoreCaseContaining(String name, String description);
```

Comment:
Can you explain why this method has such a long name and what does it do?

Reply:
Yes, sure. Please remind me the next time you see me.